pkgs/development/libraries/kerberos/heimdal.nix

```diff
@@ -76,14 +66,27 @@ stdenv.mkDerivation rec {
     (cd lib/hcrypto; make -j $NIX_BUILD_CORES)
   '';
 
+  # FIXME: share/info hits $bin, IDK why, but I decide is to minor to block
```

Comment:
You can add an `info` output for the info pages.

Reply:
info fixed